core/esmf-aspect-model-document-generators/src/test/java/org/eclipse/esmf/aspectmodel/generator/jsonschema/AspectModelJsonSchemaGeneratorTest.java

```diff
@@ -839,8 +839,8 @@ public void testEnumComplexMapping( final KnownVersion metaModelVersion ) {
             + "['properties']['description']['enum'][0]" ) ).isEqualTo( "Result succeeded" );
       assertThat( context.<String> read( "$['components']['schemas']['ResultGood']['properties']['details']"
             + "['properties']['message']['enum'][0]" ) ).isEqualTo( "Evaluation succeeded." );
-      assertThat( context.<Double> read( "$['components']['schemas']['ResultGood']['properties']['details']"
-            + "['properties']['numericCode']['enum'][0]" ) ).isCloseTo( 10.0d, Percentage.withPercentage( 1.0d ) );
+      assertThat( context.<Integer> read( "$['components']['schemas']['ResultGood']['properties']['details']"
```

Comment:
Not sure myself about this one. I wondered how this ever worked, because it was never a float/double value.

Reply:
Yes, the same question)
you are totally right..)

I found, that the build pipelines every time working like we expect, but in my local env I was getting this error...

But I did the same fix in my PR - https://github.com/eclipse-esmf/esmf-sdk/pull/441